html/pages/device/health/sensors.inc.php

```diff
@@ -16,12 +20,28 @@
     }
 
     $sensor_current = format_si($sensor['sensor_current']).$unit;
+
+    if (($graph_type == 'sensor_state') && !empty($state_translation['0']['state_descr'])) {
+        $sensor_current = get_state_label($sensor['state_generic_value'], $state_translation[0]['state_descr'] . " (".$sensor['sensor_current'].")") ; 
+    }
+
     $sensor_limit = format_si($sensor['sensor_limit']).$unit;
     $sensor_limit_low = format_si($sensor['sensor_limit_low']).$unit;
+
     echo "<div class='panel panel-default'>
-            <div class='panel-heading'>
-                <h3 class='panel-title'>$sensor_descr <div class='pull-right'>$sensor_current | $sensor_limit_low <> $sensor_limit</div></h3>
-            </div>";
+        <div class='panel-heading'>
+        <h3 class='panel-title'>$sensor_descr <div class='pull-right'>$sensor_current";
+
+    //Display low and high limit if they are not null (format_si() is changing null to '0')
+    if (!is_null($sensor['sensor_limit_low'])) {
```

Comment:
All looks except I don't like the low / high values being in red. Looks like they are triggering something rather than just informational.

Might even be worth putting them in label boxes like the current value so it's all in keeping. You could change the label colour based on it being high or low and even if it's breached the current value.

Reply:
Yep, agree. Red is not nice, it was an attempt to make it "different" than the value itself. Will try to see what I can do with label boxes.

Reply:
With a standard value, the label display is pretty nice. 
![capture d ecran 2018-12-15 a 23 54 41](https://user-images.githubusercontent.com/38363551/50048146-fc6b6f00-00c4-11e9-90b6-f1635c08a400.png)

But if you have low/high limit associated with a state, which is not forbidden, and works well (I use it in production), the display is somehow unclear : 
![capture d ecran 2018-12-15 a 23 55 32](https://user-images.githubusercontent.com/38363551/50048166-389ecf80-00c5-11e9-82bd-ae5cce084647.png)

What do you think ?

Reply:
I like the second way more but what about having orange and blue or something for high / low and then red if they are breached?

Reply:
I would say that too many colors might become difficult to read as well. 

I can try to move "current value" into label (that would be green or red depending on the conformity with min/max values)

And min/max values themselves, if availiable, would remain gray (label-default). 

But that won't be for today ;)

Reply:
Maybe instead of the the high low use arrow icons?

Reply:
Here is the variant with text labels, including warning limits, and current_value get its color based on the 4 limits. The limits themselves are label-default.

For icons, we would need to make it clear if it is limit or warn_limit, and if we are the low or high limit. I don't see "visually" how to do that. If somebody has a sketch of it, feel free. 

![capture d ecran 2018-12-16 a 12 14 54](https://user-images.githubusercontent.com/38363551/50052906-420f5280-012c-11e9-9a22-9b75638f3ee4.png)

Reply:
```
-----
 / \
/ | \
  |
```

Something like that? I couldn't find a font awesome icon that matches :(
Also tooltip text with a full description of what the field is helps too.

Reply:
I'd much prefer avoiding just using icons that rely on people hovering over to work out what they are. It's not really intuitive (we do this elsewhere in the webui and it confuses people).